chpl/chpl-service/src/main/java/gov/healthit/chpl/user/cognito/CognitoApiWrapper.java

```diff
@@ -181,9 +186,28 @@ public User getUserInfo(UUID cognitoId) throws UserRetrievalException {
             }
         }
         return null;
+    }
+
+    public User getUserByPassingCognitoUserAttributeCache(String accessToken) throws UserRetrievalException {
```

Comment:
When I first read this method name I thought it was two words "by passing" but I think it's meant as one word "bypassing" - can you change the camelcase to make this clear? getUserBypassingCognitoUserAttributeCache, or maybe even simpler getUserNoCache?